packages/contracts/test/plugins/token/distribution/merkle-distributor.ts

```diff
@@ -40,6 +41,46 @@ describe('MerkleDistributor', function () {
     distributor = await deployWithProxy(MerkleDistributor);
   });
 
+  describe('plugin interface: ', async () => {
+    it('does not support the empty interface', async () => {
+      expect(await distributor.supportsInterface('0x00000000')).to.be.eq(false);
+    });
+
+    it('supports the `IERC165Upgradeable` interface', async () => {
+      // @ts-ignore
+      const IERC165Upgradeable = await hre.artifacts.readArtifact(
+        'IERC165Upgradeable'
+      );
+      const iface = new ethers.utils.Interface(IERC165Upgradeable.abi);
+
+      expect(
+        await distributor.supportsInterface(getInterfaceID(iface))
+      ).to.be.eq(true);
+    });
+
+    it('supports the `IPlugin` interface', async () => {
```

Comment:
Why does the MerkleDistributor ship as an IPlugin?

Reply:
It is a `PluginUUPSUpgradeable` for a very long time now.

Initially (about 1 year ago) we thought that his plugin could be helpful for DAOs that want to launch their own token.
Now, it was sitting in the protocol folder as we don't have the separation between plugins / protocol yet and is not really used.